src/transform/bpcells_regress_out/script.R

```diff
@@ -0,0 +1,43 @@
+cat("Loading libraries\n")
+library(glue)
+library(BPCells)
+requireNamespace("anndata", quietly = TRUE)
+requireNamespace("reticulate", quietlye = TRUE)
+mudata <- reticulate::import("mudata")
+
+## VIASH START
+par <- list(
+    input = "resources_test/pbmc_1k_protein_v3/pbmc_1k_protein_v3_filtered_feature_bc_matrix.h5mu",
+    output = "output.h5mu",
+    modality = "rna"
+)
+## VIASH END
+
+# Read the h5mu file and make var names unique
+mdata <- mudata$read_h5mu(par$input)
+mdata$var_names_make_unique()
```

Comment:
I think the `var_names` should already be unique at this point. We could assign temporary column names to `imat` if there are still duplicate var_names.